### PR TITLE
RI-30: Add all pre-registration steps to deploy Ironic

### PR DESCRIPTION
**Pull request by prometheanfire**
This commit brings us up to the point where we can start registering nodes to
Ironic.  The actual instructions on registering nodes will be left to a later
commit.

Signed-off-by: Matthew Thode <mthode@mthode.org>

### CODE REVIEW

**Comment by cloudnull** [comment]
Are we going to cover the node image creation elsewhere?

File: IRONIC.md [hunk 1]
```diff
@@ -33,3 +109,34 @@ following variables in `` user_osa_variables_overrides.yml ``:
...
+	    automated_clean: true
+
+This will autoclean the nodes on checkin and when they are initially registered
+to Ironic.
+
+### Redeploy Ironic
+
+Rerun the `` os-ironic-install.yml `` play to get the new settings in place.
+
+## todo: Post deployment image registration and node enrollment
```
Comment (cloudnull) [inline IRONIC.md]: are we expecting this in a follow on pr or are you going to continue adding to this one?

**Comment by prometheanfire** [comment]
In a follow on commit

On July 24, 2017 4:43:48 PM CDT, Kevin Carter <notifications@github.com> wrote:
>cloudnull commented on this pull request.
>
>
>
>> +section to your `` user_osa_variables_overrides.yml ``.
>+
>+	ironic_ironic_conf_overrides:
>+	  neutron:
>+	    cleaning_network_uuid: "<NEUTRON_TFTP_NETWORK_UUID>"
>+	  conductor:
>+	    automated_clean: true
>+
>+This will autoclean the nodes on checkin and when they are initially
>registered
>+to Ironic.
>+
>+### Redeploy Ironic
>+
>+Rerun the `` os-ironic-install.yml `` play to get the new settings in
>place.
>+
>+## todo: Post deployment image registration and node enrollment
>
>are we expecting this in a follow on pr or are you going to continue
>adding to this one?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/rcbops/rpc-openstack/pull/2429#pullrequestreview-51907020

-- 
Sent from Kaiten Mail. Please excuse my brevity.

File: IRONIC.md (hunk 1 above)
Comment (prometheanfire) [inline IRONIC.md]: in a follow on commit

**Pull request merged**